docs/mkdocs.yml

```
@@ -126,7 +127,11 @@ markdown_extensions:
       pygments_lang_class: true
   - pymdownx.inlinehilite
   - pymdownx.snippets
-  - pymdownx.superfences
+  - pymdownx.superfences:
+      custom_fences:
+            - name: mermaid
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 12

(indentation)